_data/versions.yaml

```diff
@@ -7,8 +7,7 @@ maven: 3.9.6+
 documentation:
   - 'main'
   - 'latest'
+  - '3.20'
   - '3.15'
   - '3.8'
   - '3.2'
-  - '2.16'
```

Comment:
we shuoldn't do this if it means the guides urls gets removed....

Reply:
I suggest to split this in two - one to get 3.20 added ASAP
and if need to separate one figuring out how we handle archiving so we don't break doc links. we can there look at access data/stats to see how often they are accesses.

Reply:
Ok, I've put back the old versions..